test/failure_test.py

```diff
@@ -483,5 +483,34 @@ def testVersionMismatch(self):
         ray.__version__ = ray_version
 
 
+class WarningTest(unittest.TestCase):
+    def tearDown(self):
+        ray.worker.cleanup()
+
+    def testExportLargeObjects(self):
+        import ray.ray_constants as ray_constants
+
+        ray.init(num_workers=1)
+
+        large_object = np.zeros(2 * ray_constants.PICKLE_OBJECT_WARNING_SIZE)
+
+        @ray.remote
+        def f():
+            large_object
+
+        # Make sure that a warning is generated.
+        wait_for_errors(b"pickling_large_object", 1)
+
+        @ray.remote
+        class Foo(object):
+            def __init__(self):
+                large_object
+
+        Foo.remote()
+
+        # Make sure that a warning is generated.
+        wait_for_errors(b"pickling_large_object", 2)
```

Comment:
Dumb Question: is this supposed to return an error at some point? Like what happens if this test fails? There's no `unittest.assertTrue` that raises a notification of failure.

Reply:
Good catch, I'll fix that.